packages/eas-cli/src/commands/workflow/list.ts

```diff
@@ -0,0 +1,67 @@
+import EasCommand from '../../commandUtils/EasCommand';
+import { EasJsonOnlyFlag } from '../../commandUtils/flags';
+import { AppQuery } from '../../graphql/queries/AppQuery';
+import Log from '../../log';
+import formatFields from '../../utils/formatFields';
+import { enableJsonOutput, printJsonOnlyOutput } from '../../utils/json';
+
+export type WorkflowResult = {
+  id: string | null;
+  name?: string | null | undefined;
+  fileName: string | null;
+  createdAt: string | null;
+  updatedAt: string | null;
+};
+
+export default class WorkflowList extends EasCommand {
+  static override description = 'List workflows for the current project';
+
+  static override flags = {
+    ...EasJsonOnlyFlag,
+  };
+
+  static override contextDefinition = {
+    ...this.ContextOptions.ProjectId,
+    ...this.ContextOptions.LoggedIn,
+  };
+
+  async runAsync(): Promise<void> {
+    const { flags } = await this.parse(WorkflowList);
+    const {
+      projectId,
+      loggedIn: { graphqlClient },
+    } = await this.getContextAsync(WorkflowList, {
+      nonInteractive: true,
+    });
+
+    const workflows = await AppQuery.byIdWorkflowsAsync(graphqlClient, projectId);
+
+    const result: WorkflowResult[] = workflows.map(workflow => ({
+      id: workflow.id ?? null,
+      name: workflow.name ?? null,
+      fileName: workflow.fileName ?? null,
+      createdAt: workflow.createdAt ?? null,
+      updatedAt: workflow.updatedAt ?? null,
+    }));
+
+    if (flags.json) {
+      enableJsonOutput();
```

Comment:
Looking at the other commands, most tend to call this right after `getContext` in order to preempt any accidental console logs in code before the command calls `printJsonOnlyOutput`.

Prefer to do a call to it on L36 separately from the `printJsonOnlyOutput` call:
```
if (flags.json) {
  enableJsonOutput();
}
```

I think we do this as a defensive measure in case something like `AppQuery.byIdWorkflowsAsync` were to accidentally output in a future revision.